dispatcher/backend/src/periodic-tasks.py

```diff
@@ -148,6 +155,49 @@ def staled_statuses(session: so.Session):
     logger.info(f"::: failed {nb_failed_tasks} tasks")
 
 
+def tasks_cleanup(session: so.Session):
+    """removes old tasks excluding most recent tasks for schedules"""
+
+    if not SHOULD_REMOVE_OLD_TASKS:
+        logger.info(f":: skipping removal of tasks older than {TASKS_OLDER_THAN} days")
+        return
+
+    logger.info(f":: removing tasks older than {TASKS_OLDER_THAN} days")
```

Comment:
We should compute the "cutover" date and log it in this message, so that it is clear which tasks we've deleted.